server/graphql/v2/object/Order.js

```diff
@@ -202,7 +202,7 @@ export const GraphQLOrder = new GraphQLObjectType({
         type: GraphQLPaymentMethod,
         resolve(order, _, req) {
           if (order.PaymentMethodId) {
-            return req.loaders.PaymentMethod.byId.load(order.PaymentMethodId);
+            return req.loaders.PaymentMethod.byIdIncludeDeleted.load(order.PaymentMethodId);
```

Comment:
Do we really need to expose deleted assets? What is the use case?

Reply:
When looking at past contributions / transactions. If the user removes the payment method, we still want to show info about it in the contribution list/drawers.